skynet/bandwidth.go

```diff
@@ -52,10 +66,23 @@ func BandwidthDownloadCost(size int64) int64 {
 	return PriceBandwidthDownloadBase + chunks*PriceBandwidthDownloadIncrement
 }
 
+// RawStorageUsed calculates how much storage an upload with a given size
+// actually uses. This method returns the total underlying storage used and not
+// the adjusted number users see. Users see adjusted numbers in order to
+// shield them from the complexity of base/chunk redundancy.
+func RawStorageUsed(uploadSize int64) int64 {
+	baseSectorStorage := int64(PriceStorageUploadBase * RedundancyBaseSector)
```

Comment:
Why are we multiplying a storage price with a redundancy to get a storage amount?

Reply:
Hm, this might be more of a naming issue than a real problem.

The basic premise here is that we charge everything in bytes, so a registry access "costs" 1MiB for a read and 5MiB for a write operation. That's why all of these are prefixed by "Price" - it's more to group them together logically than to add markup in any way.

The goal of this logic is to calculate how much actual storage a given upload takes up.  
So, `PriceStorageUploadBase` is the amount of data a base sector on its own takes up - 4MiB. `PriceStorageUploadIncrement` is how much a data chunk takes up - 40MiB.

I'm not sure this clear things up enough but I'm not sure how to approach this in a comment. Maybe we can try it real-time (voice or text) and figure out a better approach.

Reply:
Renamed all Price* constants to Cost* in order to capture the idea that they represent bandwidth/storage costs rather than monetary values.